clap_man/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "clap_man"
+version = "0.1.0"
+edition = "2018"
+include = [
+	"src/**/*",
+	"Cargo.toml",
+	"LICENSE-*",
+	"README.md"
+]
+description = "A manpage generator for clap"
+repository = "https://github.com/clap-rs/clap/tree/master/clap_man"
+documentation = "https://docs.rs/clap_man"
+keywords = [
+	"clap",
+	"cli",
+	"generate",
+	"manpage",
+]
+categories = ["command-line-interface"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+
+[package.metadata.release]
+pre-release-replacements = [
+  {file="README.md", search="github.com/clap-rs/clap/blob/[^/]+/", replace="github.com/clap-rs/clap/blob/{{tag_name}}/", exactly=4, prerelease = true},
+]
```

Comment:
I added a changelog + changelog updating to `clap_complete` (37f47dd2600fc55b3a367249d71918277e373bb6).  Its a bit specific of a process, so if you want, I can do it once this is merged.

Reply:
Would probably be better if you did it.